SW/main/Thermal.py

```diff
@@ -0,0 +1,124 @@
+from Adafruit_AMG88xx import Adafruit_AMG88xx
+import Stove
+import pygame
+import os
+import math
+import time
+from time import sleep
+
+import numpy as np
+from scipy.interpolate import griddata
+
+from colour import Color
+
+class Thermal:
```

Comment:
can you rename this ThermalMeasurer to make it more specific (or anything along those lines)